### DISCUSSION TITLE
Any testers out there? [x265 Progress Readframes]

### DISCUSSION THREAD

**Discussion by Dendraspis** [staxrip/staxrip#509] [Show and tell]
Hey @44vince44 and @JJKylee ,
hello everybody,

are you interested in testing a new feature? 😉

This time just a very small one, but in certain situations helpful:
I implemented the new `--progress-readframes` parameter for @DJATOM's builds since [`3.4+70-aMod`](https://github.com/DJATOM/x265-aMod/releases/tag/3.4%2B70). To not interfere with other builds, this feature is only available when using a DJATOM build, but regardless its version number!

If you wanna test it:
[StaxRip v2.1.8.0D_x265.zip](https://github.com/staxrip/staxrip/files/5968476/StaxRip.v2.1.8.0D_x265.zip)

<img width=600 src="https://user-images.githubusercontent.com/50659978/107699618-9daadb80-6cb6-11eb-845f-636a5dcbc426.PNG">

**Comment by JJKylee** [comment]
Will do when I get back to my computer in about a couple hours. But I'm pretty sure there will be no complicated issues. 😄

**Reply by Dendraspis** [reply]
Maybe testing it with another x265 build, just to be sure the verification works, that I haven't tested in detail.  😊  

*Honestly I created this discussion to let you and @44vince44 know about this possibility, because I made this improvement especially for both of you.*   😁

**Reply by JJKylee** [reply]
It _does_ work as you described.

![StaxRip v2 1 8 0D_x265_DJATOM's_build_20210211](https://user-images.githubusercontent.com/3157518/107726517-0fc2f680-6c9e-11eb-93db-daeafa8cab85.jpg)

![StaxRip v2 1 8 0D_x265_Patman's_build_20210211](https://user-images.githubusercontent.com/3157518/107726529-16ea0480-6c9e-11eb-8299-4e9bd412fea3.jpg)

Good job! 👍👏

**Reply by ghost** [reply]
I'm testin it... what does it do ?
Ah it adds the read frames vs the processed one, right ?
Progress:
Processed_frames (Read_Frames) / Total_Frames_to_process

Is that right ? Well it works 😄

**Reply by Dendraspis** [reply]
Yes, that's right.  😄 

Maybe an interesting information for debugging?!

**Reply by ghost** [reply]
Yes i guess so, it would give information about read errors... Since x265 is very clean unlike other codecs I won't mention, I doubt it will be useful 😁

**Reply by Dendraspis** [reply]
I think of slow source drives, filter issues, ...
Let's see, but you might be right.  😁

**Reply by ghost** [reply]
@Dendraspis shouldn't the option be in the Statistics section and not in Other2 ?

**Reply by Dendraspis** [reply]
Why not ..... done.  😁
